### PR TITLE
Simplify the validate_RESULT function

### PR DESCRIPTION
**Pull request by poncovka**
Move the code to other functions.

### CODE REVIEW

**Review by rvykydal — APPROVED** [review]
Looks good to me.

File: functions.sh [hunk 1]
```diff
@@ -124,20 +131,18 @@ copy_interesting_files_from_system() {
 }
 
 validate_RESULT() {
-    disksdir=$1
```
Comment (VladimirSlavik) [inline functions.sh:127]: I think having it as a local named variable might have been useful, but never mind.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

File: functions.sh (hunk 1 above)
Comment (jkonecny12) [reply functions.sh:127]: +1 it's more readable to set a name for the variable.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me aside the Vlada's comment.

**Review by M4rtinK — APPROVED** [review]
Looks good to me, nice cleanup!

**Pull request merged**